src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitations/GetInvitationsQueryHandler.cs

```diff
@@ -31,20 +32,58 @@ public async Task<Result<InvitationsResult>> Handle(GetInvitationsQuery request,
             // count before adding sorting/paging
             var maxAvailable = await queryable.CountAsync(token);
 
-            var enumerable = AddSorting(request.Sorting, queryable.AsEnumerable());
-            enumerable = AddPaging(request.Paging, enumerable);
+            queryable = AddSorting(request.Sorting, queryable);
+            queryable = AddPaging(request.Paging, queryable);
 
-            var orderedDtos = enumerable.ToList();
+            var orderedDtos = await queryable.ToListAsync(token);
```

Comment:
I think we can remove this line as the Any() call below will execute the query?

Reply:
I think we need it as List/IEnumerable in CreateResult

Reply:
But I can do ToList in/right before CreateResult() if that is better?